engine/proofview.mli

```diff
@@ -461,15 +462,6 @@ module Unsafe : sig
   (** [tclGETGOALS] returns the list of goals under focus. *)
   val tclGETGOALS : Proofview_monad.goal_with_state list tactic
 
-  (** [tclSETSHELF gls] sets goals [gls] as the current shelf. *)
-  val tclSETSHELF : Evar.t list -> unit tactic
-
-  (** [tclGETSHELF] returns the list of goals on the shelf. *)
-  val tclGETSHELF : Evar.t list tactic
```

Comment:
That one might still be useful as a high-level tactic, the other commit only adds a way to push stuff on the shelve.

Reply:
I put it back.